src/System.CommandLine.Tests/Binding/TypeConversionTests.cs

```diff
@@ -206,10 +213,23 @@ public void Argument_parses_as_the_default_value_when_the_option_has_not_been_ap
         [Fact]
         public void Argument_does_not_parse_as_the_default_value_when_the_option_has_been_applied()
         {
-            var command = new Command("something", "", new[] {
-                new Option("-x", "",
-                           new Argument<int>(123))
-            });
+            IReadOnlyCollection<Symbol> symbols = new[] {
+                new Option("-x")
+                {
+                    Argument = new Argument<int>(() => 123)
+                }
+            };
+            var command1 = new Command(
+                "something",
+                ""
```

Comment:
Is the empty description needed here?

Reply:
No, this is an artifact of this having been previously required.